spec/client/ics-008-wasm-client/README.md

```diff
@@ -119,7 +115,7 @@ interface Height {
 
 ### Headers
 
-Contents of Wasm client headers depend upon `Wasm Contract`.
+Contents of Wasm client headers depend upon Wasm contract. Binary data represented by the `data` field is opaque and only interpreted by the Wasm contract.
```

Comment:
Would "contract" be the proper term? We can instead choose a more general term like WASM "program".